docs/guide/agent-server-compatibility.asciidoc

```diff
@@ -0,0 +1,75 @@
+[[agent-server-compatibility]]
+== Agent/Server Compatibility
+
+Below is a chart that outlines the compatibility between different versions of the APM agents and the APM Server. 
+
+NOTE: APM Server version 6.5 introduced a new way for APM agents to communicate with the APM Server.
+To take advantage of new features,
+it is recommended that you update to the newest version of the applicable agent.
+
+[float]
+[[nodejs-compatibility]]
+==== Node.js Agent Compatibility
+
+[options="header"]
+|=======================================================================
+|Agent Version |APM Server Version
+|1.x |6.2.0-6.5.0
```

Comment:
It seems from this range (6.2.0-6.5.0) that agent v1.x supports server v6.5.0. That's not the case however. It only supports 6.2-6.4. I think if you drop the zero it indicates that it's the entire 6.4-range. You could also add an "x", i.e. 6.2.x-6.4.x 🤷‍♂️

Reply:
Some of the other ranges for the other agents below suffer from the same ambiguity

Reply:
You're right, I had a bunch of mistakes in the server compatibility. Pushing fixes.